### PR TITLE
Change large vehicle controller parameter

### PR DESCRIPTION
**Pull request by iman512003**
This is to address #199 .

### CODE REVIEW

File: smarts/core/tests/test_trajectory_controller.py [hunk 1]
```diff
@@ -19,11 +21,28 @@
...
 
 
+@pytest.fixture(params=["sedan", "bus"])
+def vehicle_controller_file(request):
+    vehicle_file_name = request.param + ".urdf"
+    if request.param == "sedan":
+        vehicle_file_name = "vehicle.urdf"
+
+    with pkg_resources.path(models, vehicle_file_name) as path:
+        VEHICLE_FILEPATH = str(path.absolute())
```
Comment (JVillella) [inline smarts/core/tests/test_trajectory_controller.py:37]: please use lowercase (snake case)

File: smarts/core/tests/test_trajectory_controller.py [hunk 2]
```diff
@@ -40,13 +59,18 @@ def bullet_client(TIMESTEP_SEC=time_step):
 
 
 @pytest.fixture
-def vehicle(bullet_client, TIMESTEP_SEC=time_step):
+def vehicle(bullet_client, vehicle_controller_file, TIMESTEP_SEC=time_step):
```
Comment (JVillella) [inline smarts/core/tests/test_trajectory_controller.py:62]: I don't see where it's been updated to pass these new params in?

**Review by JVillella — CHANGES_REQUESTED** [review]
See inline comments.

File: smarts/core/tests/test_trajectory_controller.py (hunk 2 above)
Comment (iman512003) [reply smarts/core/tests/test_trajectory_controller.py:62]: Thanks, it is inside the `controller_parameters.yaml` inside the following lines:

```
with pkg_resources.path(models, "controller_parameters.yaml") as controller_path:
        controller_filepath = str(controller_path.absolute())
    with open(controller_filepath, "r") as controller_file:
        VEHICLE_CONTROLLER_PARAMETERS = yaml.safe_load(controller_file)[request.param]
```

File: smarts/core/tests/test_trajectory_controller.py (hunk 1 above)
Comment (iman512003) [reply smarts/core/tests/test_trajectory_controller.py:37]: Thanks, sure.

File: smarts/core/tests/test_trajectory_controller.py (hunk 2 above)
Comment (JVillella) [reply smarts/core/tests/test_trajectory_controller.py:62]: I mean where you call vehicle(...)

**Review by JVillella — APPROVED** [review]

File: smarts/core/tests/test_trajectory_controller.py (hunk 2 above)
Comment (liamchzh) [reply smarts/core/tests/test_trajectory_controller.py:62]: `vehicle_controller_file` is also a fixture.

**Review by liamchzh — APPROVED** [review]
Notice this test file has been using `TIMESTEP_SEC`. Probably rename them to lowercase for better code style. Others LGTM.

**Pull request merged**